Linux X86_64 wheels (#14)
PennyLaneAI/pennylane-lightning-kokkos: .github/CHANGELOG.md

> # Release 0.27.0

Reviewer: We didn't have a version bump for 0.28 dev version.
Could you please use this PR to make that?

Author: Good idea! I plan to work on it in a separate PR to make the updates organizing in more clear way. How do you think?

Reviewer: Feel free to open a new PR for that, but I think you can do it here if you like.

Author: done! Thanks

# Release 0.28.0

 
### New features since last release

 * Add X86-64 Linux wheels building with OpenMP backend for lightning.kokkos in workflows.
 [(#14)](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/14)

### Breaking changes

> 

Reviewer: ```suggestion

 * Remove support for Python 3.7
 [(#14)](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/14)
 
```

 * Deprecate the Python and C++ tests with threading backend in workflows.
 Note this deprecation is based on the fact that Kokkos cannot promise that its Threads back-end will 
 not conflict with the application's direct use of operating system threads. 

>  [(#14)](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/14)

Reviewer: ```suggestion
 [(#14)](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/14)
 
```

### Improvements

> 

Reviewer: ```suggestion

* Remove the unused `supports_reversible_diff` device capability from `LightningKokkos`
 [(#14)](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/14)
 
```

### Documentation

### Bug fixes

### Contributors

This release contains contributions from (in alphabetical order):

> Shuli Shu

Reviewer: ```suggestion
Shuli Shu, Matthew Silverman
```

# Release 0.28.0

 
### New features since last release

 * Add X86-64 Linux wheels building with OpenMP backend for lightning.kokkos in workflows.
 [(#14)](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/14)

### Breaking changes
 * Add built information query support with the `_built_backend_info` method.
 The workflow for `_built_backend_info`is:
 ```python
 dev = qml.device('lightning.kokkos', wires=3)

>  dev._built_backend_info()

Author: done!

>  dev._built_backend_info()

Reviewer 2: This must've been a carry-over from an older review I never deleted. Feel free to ignore.

### New features since last release

>  * Add X86-64 Linux wheels building with OpenMP and SERIAL backend for lightning.kokkos in workflows.

Reviewer 2: ```suggestion
 * Add support for building X86-64 Linux wheels with OpenMP and SERIAL backends with Github Actions.
```

### New features since last release

 * Add X86-64 Linux wheels building with OpenMP backend for lightning.kokkos in workflows.
 [(#14)](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/14)

### Breaking changes

>  * Add the build information record support by adding the `record_build_info()` method in `setup.py`.

Author: Let me move it to the new feature section.

>  * Add the build information record support by adding the `record_build_info()` method in `setup.py`.

Author: done

 With the `record_build_info()` method, running before the `setup()` method, a json file `build_info.json` will be created and stored with the build information of backend, device architecture and platform. This json file is then included into the installed package. 
 [(#14)](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/14)

 * Add the build information query support with the `_build_info(keyname)` method.
 Supported `keyname`s are `Backend`, `Device_Arch` and `Platform`.
 The workflow for `_build_info(keyname)`is:
 ```python
 >>> import pennylane as qml
 >>> import pennylane_lightning_kokkos as plk
 >>> dev = qml.device('lightning.kokkos', wires=3)
 >>> dev._build_info('Backend')
 "OPENMP"

>  ```

Author: Let me move it. Thanks
